lib/ansiblelint/runner.py

```diff
@@ -0,0 +1,94 @@
+"""Runner Module."""
```

Comment:
It should describe that's inside, not just recite the file name and things that are already known. This is hard but we could do at least a bit better:
```suggestion
"""Runner implementation."""
```